utilities/write_batch_with_index/write_batch_with_index.cc

```diff
@@ -487,6 +487,32 @@ Status WriteBatchWithIndex::GetFromBatch(ColumnFamilyHandle* column_family,
   return s;
 }
 
+Status WriteBatchWithIndex::GetEntityFromBatch(
+    ColumnFamilyHandle* column_family, const Slice& key,
+    PinnableWideColumns* columns) {
+  assert(columns);
```

Comment:
It would be good be consistent with checks with the GetEntity API. This would also save our users from making mistakes :)

```
if (!column_family) {
  return Status::InvalidArgument(
      "Cannot call GetEntityFromBatch without a column family handle");
}
if (!columns) {
  return Status::InvalidArgument(
      "Cannot call GetEntityFromBatch without a PinnableWideColumns object");
}
```

Reply:
Hmm... I just found that `WriteBatchWithIndex::GetFromBatchAndDB` and `MultiGetFromBatchAndDB` use the default CF when CFH is provided as nullptr.

I believe we may want to expose APIs without CFH parameter, which will internally call the same API with the default CFH. In this way, users should either provide a non-null CFH pointer or call the API that doesn't take a CFH at all. I think this is the general pattern for our APIs in db.h.

No strong opinion on this. And we can address this in a separate PR, too.